test_suite.sh

```diff
@@ -217,11 +254,25 @@ else
     fatal_error "Failed to extract names of tests to run: ${REFRAME_NAME_ARGS}"
     exit ${test_selection_exit_code}
 fi
-export REFRAME_ARGS="--tag CI --tag 1_node --nocolor ${REFRAME_NAME_ARGS}"
+# Allow people deploying the bot to overrwide this
+if [ -z "$REFRAME_SCALE_TAG" ]; then
+    REFRAME_SCALE_TAG=--tag=1_4_node
+fi
+if [ -z "$REFRAME_CI_TAG" ]; then
+    REFRAME_CI_TAG=--tag=CI
+fi
+# Allow bot-deployers to add additional args through the environment
+if [ -z "$REFRAME_ADDITIONAL_ARGS" ]; then
+    REFRAME_ADDITIONAL_ARGS=""
+fi
+export REFRAME_ARGS="${REFRAME_CI_TAG} ${REFRAME_SCALE_TAG} ${REFRAME_ADDITIONAL_ARGS} --nocolor ${REFRAME_NAME_ARGS}"
 
 # List the tests we want to run
 echo "Listing tests: reframe ${REFRAME_ARGS} --list"
-reframe ${REFRAME_ARGS} --list
+echo ${REFRAME_ARGS}
+echo "${REFRAME_ARGS}"
+echo $REFRAME_ARGS
+reframe ${REFRAME_CI_TAG} ${REFRAME_SCALE_TAG} ${REFRAME_ADDITIONAL_ARGS} --nocolor "${REFRAME_NAME_ARGS}" --list -v
```

Comment:
wait, so... there was no way to get `reframe ${REFRAME_ARGS}` to work (with _any_ choice of quotes/braces)? Keeping that in a single environment variable was done to ensure that `--list` and `--run` would be called with the exact same arguments... The current way of doing things opens up the possibility of someone adding something to the argument list when calling `--list` but forgetting to do so for the `--run`.